pkgs/applications/video/bilibili/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchurl
+, electron
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bilibili";
+  version = "1.9.2-1";
+  src = fetchurl {
+    url = "https://github.com/msojocs/bilibili-linux/releases/download/v${version}/io.github.msojocs.bilibili_${version}_amd64.deb";
+    hash = "sha256-y3dUBImvcIG89m82RaIOa0cxJXIAIGa+n3FJkASacaY=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    ar x $src
+    tar xf data.tar.xz
+
+    runHook postUnpack
+  '';
+ 
+  nativeBuildInputs = [ makeWrapper ];
```

Comment:
```suggestion
  '';

  nativeBuildInputs = [ makeWrapper ];
```

Reply:
there are no whitespace displayed in my editor, so strange